_includes/start_in_docker/v2.0/mac-linux-steps.md

```diff
@@ -0,0 +1,167 @@
+## Before You Begin
```

Comment:
It seems silly to duplicate this entire file based on the version that's being run, but I take it there wasn't a good way to share it?

Reply:
You're right. I put in place a global variable. PTAL.